packages/libs/error-stack/src/context.rs

```diff
@@ -1,15 +1,19 @@
 #[cfg(nightly)]
 use core::any::Demand;
+#[cfg(nightly)]
+use core::error::Error;
 use core::fmt;
+#[cfg(all(not(nightly), feature = "std"))]
+use std::error::Error;
 
 use crate::Report;
 
 /// Defines the current context of a [`Report`].
 ///
-/// When in a `std` environment, every [`Error`] is a valid `Context`. This trait is not limited to
-/// [`Error`]s and can also be manually implemented for custom objects.
+/// When in a `std` environment or on a nightly toolchain, every [`Error`] is a valid `Context`.
+/// This trait is not limited to [`Error`]s and can also be manually implemented for custom objects.
```

Comment:
```suggestion
/// This trait is not limited to [`Error`]s and can also be manually implemented on a type.
```

This would be inline with the official rust docs wording: https://doc.rust-lang.org/book/ch10-02-traits.html#implementing-a-trait-on-a-type

Reply:
Good point, thanks! 🙂